ExampleScript.py

```diff
@@ -148,6 +151,6 @@
 # Create stellarator
 parastell.parastell(
     plas_eq, num_periods, build, gen_periods, num_phi, num_theta,
-    magnets = magnets, source = source,
```

Comment:
Perhaps you turned off the magnets and source for testing, but I think we should turn them back as we merge this PR.

Reply:
agreed